build.rs

```diff
@@ -134,10 +187,149 @@ fn fix_config_map(mut config_map: HashMap<String, String>) -> HashMap<String, St
     config_map
 }
 
-fn load_cross_compile_info() -> Result<(InterpreterConfig, HashMap<String, String>)> {
-    let python_include_dir = env::var("PYO3_CROSS_INCLUDE_DIR")?;
-    let python_include_dir = Path::new(&python_include_dir);
+fn parse_script_output(output: &str) -> HashMap<String, String> {
+    output
+        .lines()
+        .filter_map(|line| {
+            let mut i = line.splitn(2, ' ');
+            Some((i.next()?.into(), i.next()?.into()))
+        })
+        .collect()
+}
+
+fn as_bool(config: &HashMap<String, String>, key: &str) -> Result<bool> {
```

Comment:
I think `.as` in the stdlib is typically used for zero-cost coercions.

```suggestion
fn get_bool(config: &HashMap<String, String>, key: &str) -> Result<bool> {
```

Reply:
You're right, folllowing the [api guidelines][1] it should probably be `to_` or `into_` but would make more sense to implement as a method on `Option<String>` in our own trait.

[1]: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Reply:
But if its kept as an accessor, get effectively more accurate.